components/elements/input/__test__/input.test.tsx

```diff
@@ -0,0 +1,35 @@
+/** @jest-environment jsdom */
+import {render, screen, fireEvent} from '@testing-library/react';
+import "@testing-library/jest-dom"
+import Input from "../input";
+import {ThemeProvider} from "styled-components";
+import theme from "../../../../lib/theme";
+
+
+describe("input", () => {
+
```

Comment:
1. use formatter.
2. new function for render.
3. update test names.
4. new line.